src/Raven.Client/Documents/Session/ClusterTransactionSession.cs

```diff
@@ -240,6 +240,9 @@ internal CompareExchangeSessionValue RegisterCompareExchangeValue(CompareExchang
         {
             Debug.Assert(value != null, "value != null");
 
+            if (value.Key.StartsWith(Constants.CompareExchange.RvnAtomicPrefix))
```

Comment:
Should do a case insensitive check here.

Reply:
done